djangocms_versioning/plugin_rendering.py

```diff
@@ -24,14 +27,33 @@ def prefetch_versioned_related_objects(instance, toolbar):
             qs = versionable.content_model.objects.all()
         related_field = getattr(instance, field.name)
         if related_field:
-            filters = {versionable.grouper_field_name: related_field}
+            is_related_manager = isinstance(related_field, models.Manager)
+
+            if is_related_manager:
+                filter_key = '{}__in'.format(versionable.grouper_field_name)
+                filter_values = related_field.all()
```

Comment:
I'm not sure this is the best way to prefetch a relationship with .all() because any filters added to the related values will then need to discard anything from this query adding overhead to this query and then having to call other queries anyway.

Reply:
@Aiky30  @jonathan-s ,
Can you give me some advice on it. how to handle this?

Reply:
Instead of using `all()` it's probably better to use `get_queryset()` that should preserve any filters that had been added prior to this.

Reply:
@Aiky30 , @jonathan-s ,
Yes, I tried Jonathan's solution and it works.